compiler/rustc_error_messages/locales/en-US/parser.ftl

```diff
@@ -153,3 +153,6 @@ parser_left_arrow_operator = unexpected token: `<-`
 
 parser_remove_let = expected pattern, found `let`
     .suggestion = remove the unnecessary `let` keyword
+
+parser_use_eq_instead = unexpected `==`
+    .suggestion = replace `==` with a equal symbol
```

Comment:
I think this should be "with `=`" -- it won't be so verbose if you use suggestion_short